src/apprt/gtk/gresource.zig

```diff
@@ -121,12 +134,18 @@ pub fn main() !void {
         \\  <gresource prefix="/com/mitchellh/ghostty/icons">
         \\
     );
-    for (icons) |icon| {
+    for (ghostty_icons) |icon| {
         try writer.print(
             "    <file alias=\"{s}/apps/com.mitchellh.ghostty.png\">images/icons/icon_{s}.png</file>\n",
             .{ icon.alias, icon.source },
         );
     }
+    for (numeric_icons) |icon| {
+        try writer.print(
+            "    <file alias=\"scalable/status/numeric-{0s}-circle-outline-symbolic.svg\">images/icons/numeric-{0s}-circle-outline-symbolic.svg</file>\n",
+            .{icon},
+        );
+    }
```

Comment:
Why not: (you could extract the 10 out to another variable if you want)
```suggestion
    for (0..10) |idx| {
        try writer.print(
            "    <file alias=\"scalable/status/numeric-{0}-circle-outline-symbolic.svg\">images/icons/numeric-{0}-circle-outline-symbolic.svg</file>\n",
            .{idx + 1},
        );
    }
```

Reply:
Actually, after doing a bit more research, I _think_ that we can support generating the SVGs even back to Debian Bookworm. Difficult to verify 100%. I just opted to drop the static SVGs altogether.